### ISSUE TITLE
Custom controls?

### ISSUE THREAD

**Issue by francoism90** [zcuric/plyrue#10]
**Is your feature request related to a problem? Please describe.**

At the moment I've simple disabled the Plyr controls and using a component with a z-index hack on top of the video instance.

**Describe the solution you'd like**

It would be great if one could inject a Vue component as customization would be a lot easier.

**Describe alternatives you've considered**

Current workaround works fine. I'm using a bus to handle events and the 'Controls' component simple sends them back to the player.

**Additional context**

Custom controls are possible, but I don't believe with Vue components. Please correct me when I'm wrong and if possible show me how it's done. :)

Thanks for your work!

**Comment by zcuric** [comment]
From what I read on `plyr` documentation it is possible to add [custom controls like this](https://github.com/sampotts/plyr/blob/master/controls.md#example-1).

So you would have to add a `controls` field in `options` when setting up Plyrue. This could be done easier to add `controls` in `pluginOptions` when Plyrue is initialised by `vue` so you don't have to define it everywhere. 

From the documentation above, if HTML can be set to `controls` field as a function, maybe `vue` component could be injected. I could try, or you can if you have time. 

Some examples: 
https://codepen.io/jwjcmw/pen/pedQRL
http://jsfiddle.net/onigetoc/ehqu4o6x/

**Comment by francoism90** [comment]
@zcuric Thanks for your reply!

I'm indeed aware of using the `controls` options, but as far I've tested it isn't possible to inject a (native) Vue component, only plain HTML seems to be possible. However I haven't fully tested other things, like tweaking the project/app.js/.. configuration.

At the moment I've simply disabled the controls, and use a custom Vue component that hovers on top of the Plyr-instance (including fullscreen), which seems to work pretty well. This also makes connecting to other stuff like Vuex and (template) framework a lot easier.

Maybe moving this to the actual controls function would help Plyr(eu) integration. Will try to investigate when possible.
